tests/contrib/names/test_names_resource.py

```diff
@@ -85,3 +144,33 @@ def test_names_resolve(client, example_name, h, prefix):
 
     res = client.get(f"{prefix}/orcid/0000-0002-5082-6404", headers=h)
     assert res.status_code == 404
+
+
+def test_names_suggest_sort(client, example_multiple_names, h, prefix):
+    """Test a successful search."""
+
+    # With typo
+    res = client.get(f"{prefix}?suggest=davisson", headers=h)
+    assert res.status_code == 200
+    assert res.json["hits"]["total"] == 1
+    assert res.json["hits"]["hits"][0]["name"] == "Davidson, Niels John"
+
+    # With accent
+    res = client.get(f"{prefix}?suggest=dwayne", headers=h)
+    assert res.status_code == 200
+    assert res.json["hits"]["total"] == 1
+    assert res.json["hits"]["hits"][0]["name"] == "Johnsœn, Dwäyne"
```

Comment:
Can you also add a test for `?suggest=Dwäyne`, i.e capitalized and accent? I guess it should return the same results.